PowerEditor/src/Parameters.h

```diff
@@ -239,6 +239,8 @@ struct sessionFileInfo : public Position
 	int _individualTabColour = -1;
 	bool _isRTL = false;
 	bool _isPinned = false;
+	std::wstring _originalTabName;

```

Comment:
Why do we need to remeber _originalTabName here?
If the feature "use 1st line for tab name" is disnabled, it's useless completely.
Otherwise, we can use the current tab name (on Notepad++ starup time) as the original tab name.

Reply:
Now I'm wondering if restoring original tab name is really needed...
Let's make this feature even simpler.
If this feature is ON and tab's never renamed, the tab name will be modified according the first line. But if the first line is empty (for example, the 1st line becomes the 2nd one because of keystroke of ENTER; or all content are removed), then tab name just remains as it is. 
If user is not happy about it, he/she can always rename the tab manually, then the tab name cannot be modified automatically anymore.
Is there any case I'm missing?